### ISSUE TITLE
Add your name to the contributors list!

### ISSUE THREAD

**Issue by Looky1173** [Looky1173/Itinerary-Contributors#1]
**Have you contributed to the development of Itinerary, whether it's code, documentation or even an idea?** If so, you are eligible to have your name added to the list of contributors and be listed [on Itinerary's website](https://itinerary.eu.org/about).

We are following the all-contributors specification, and this repository contains the list of contributors who worked on any Itinerary repositories (those are: [Itinerary](https://github.com/Looky1173/Itinerary), [Itinerary-Server](https://github.com/Looky1173/Itinerary-Server), and [Itinerary-Contributors](https://github.com/Looky1173/Itinerary-Contributors)).

<hr>

To be added, **comment on this issue mentioning @all-contributors like the example below** (_You don't need to follow the template exactly._):

```
@all-contributors add <username> for <contributions>
```

`<contributions>` can be one or a combination of the following [emoji keys](https://allcontributors.org/docs/en/emoji-key).

For example:
```
@all-contributors add @Looky1173 for bug
@all-contributors please add @Looky1173 for code, design, and infra
@all-contributors can you add @Looky1173 for doc?
```
[Learn more...](https://allcontributors.org/docs/en/bot/usage)

<hr>

Please use **only** the comment section of this issue below to add yourself!

**Comment by Looky1173** [comment]
@all-contributors please add @Looky1173 for code, design, ideas, and infra

**Comment by Looky1173** [comment]
@all-contributors please add @Mylan1173 for userTesting

**Comment by Looky1173** [comment]
@all-contributors please add @Mylan1173 for ideas

**Comment by Looky1173** [comment]
@all-contributors please add @Chiroyce1 for ideas and testing

**Comment by Looky1173** [comment]
@all-contributors please add @mxmou for code

**Comment by Looky1173** [comment]
@all-contributors please add @AnAlgorithm for ideas and code

**Comment by Looky1173** [comment]
@all-contributors please add @kccuber-scratch for testing and filing bug reports

**Comment by Looky1173** [comment]
@all-contributors please add @AnAlgorithm for filing bug reports

**Comment by Looky1173** [comment]
@all-contributors please add @Polygon for ideas

**Comment by polygon** [comment]
I don't know what this project is and I don't know why I am repeatedly mentioned here or added as a contributor. Please remove me from all contributor lists and any instances where I am was mistakenly put on. Thank you for your cooperation.

**Comment by Looky1173** [comment]
> I don't know what this project is and I don't know why I am repeatedly mentioned here or added as a contributor. Please remove me from all contributor lists and any instances where I am was mistakenly put on. Thank you for your cooperation.

@polygon Sorry, I've just realised my mistake! I wanted to add @PoIygon, with a capital "i". I've reverted the pull request.

**Comment by Looky1173** [comment]
https://github.com/all-contributors please add @PoIygon for ideas

**Comment by Looky1173** [comment]
@all-contributors please add @PoIygon for ideas

**Comment by polygon** [comment]
It's alright, I can see how that is confusing :)

**Comment by Sid72020123** [comment]
Can I be added? I only reported bugs and tested some things and nothing more.
It's your choice to add me.

**Comment by Looky1173** [comment]
> Can I be added? I only reported bugs and tested some things and nothing more.
> It's your choice to add me.

Of course! Sorry for forgetting you; it’s hard to keep track of who contributed what and when.

@all-contributors please add @Sid72020123 for testing and filing bug reports

**Comment by Looky1173** [comment]
@all-contributors please add @Sid72020123 for testing and filing bug reports
